src/Assets/LightSkyDefense/Scripts/Enemies/Enemy.cs

```diff
@@ -39,9 +66,22 @@ private void Start()
     {
         _energyCharge = EnergyCapacity;
         _pathPoints = GameManager.Instance.Path.PathPoints;
+        _scoreboard = GameObject.Find("Scoreboard").GetComponent<Scoreboard>();
```

Comment:
The scoreboard isn't referenced in this file afaik.

Reply:
fixed. ty